Level-Editor.md

```diff
@@ -14,15 +24,26 @@ Videos:
 - [Advanced Tutorial (Part 2/2)](https://www.youtube.com/watch?v=UoaGDuBax6E) by Oliver Buo
 - [Expert Tutorial](https://www.youtube.com/watch?v=lL3oZbPfw08) by Oliver Buo
 
+---
+
 ## Sharing Levels
 
-When you finished creating a level or even more, it is possible to share it with the SuperTux community.
+When you finished creating a level or even more, it is possible to share your creations with the SuperTux community.
+
+Upload your work to the SuperTux Forum or share it on our Discord Sever. You can also create your own Add-on for the
+game by submitting it to our Add-on Manager!
+
+You can find the respective links [here](https://github.com/SuperTux/supertux/wiki#for-level-designer).
+
+### Getting The Relevant Files
```

Comment:
Maybe the information about packaging add-ons, listed from here to the end of this file, should be merged with the one in `Add-ons.md`, contained in the "Manually packaging addons" section? Pull request #35 makes updates to that file, so it's relatively more updated and can be used again. There could be a link here, which will redirect the user to that specific paragraph in `Add-ons.md`.